### ISSUE TITLE
License field doesn't actually display on full results page

### ISSUE THREAD

**Issue by dolsysmith** [gwu-libraries/scholarspace-hyrax#458]
...even when a value has been set and is visible when editing the work. Not sure if a decision was made at some point to hide the field, but I'm not sure what the point would be of a hidden metadata field that is also a required field.

**Comment by kerchner** [comment]
@dolsysmith Sounds good.  Is someone taking responsibility for finding and proposing viable substitutes?

**Comment by dolsysmith** [comment]
I'll ask Barrett and Dalton to follow up on this.

**Comment by dolsysmith** [comment]
Schol Comm's preference would be as follows:
1. Display the license field on all works in the UI.
2. Replace the current `id` value for the `All rights reserved` (which now leads to a page explaining that the term has been retired by Europeana) with the literal text or some other identifier. 
3. When the value of the license field is `All rights reserved`, the text should *not* appear as a hyperlink.

Basically, we still want `All right reserved` to display when that is the license value that has been applied, but there is currently nothing to link to in association with that value.
